fenix/app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -327,6 +329,10 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
                     Events.appOpened.record(Events.AppOpenedExtra(it))
                     // This will record an event in Nimbus' internal event store. Used for behavioral targeting
                     components.analytics.experiments.recordEvent("app_opened")
+
+                    if (safeIntent.action.equals(ACTION_OPEN_PRIVATE_TAB)) {
+                        AppIcon.newPrivateTabTapped.record(NoExtras())
```

Comment:
The metrics above `appOpened` states that there are three ways the app is opened:  `app_icon`, `custom_tab` or `link`.  In this case should we check the intent source to make sure it's `app_icon` before recording `AppIcon.newPrivateTabTapped.record(NoExtras())`?